docs/manage/data/query.md

```diff
@@ -0,0 +1,128 @@
+
+## Query tabular data directly from a compatible client
+
+Once you have synced tabular data to the Viam app, you can directly query that data from an MQL-compatible client, such as [`mongosh`](https://www.mongodb.com/docs/mongodb-shell/) or [Compass](https://www.mongodb.com/docs/compass/current/).
+
+1. If you haven't already, [configure direct data query](/services/data/configure-data-query/#configure-data-query) to be able to use this query mode.
+
+1. Open your chosen MQL-compatible client an connect to the Viam organization's MongoDB Atlas instance.
+   You can use any client that is capable of connecting to a MongoDB Atlas instance, including the [`mongosh` shell](https://www.mongodb.com/docs/mongodb-shell/), [MongoDB Compass](https://www.mongodb.com/docs/compass/current/), and many third-party tools.
+   To connect, use the `hostname` you determined when configuring direct data query, and structure your username in the following format:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   db-user-<YOUR-ORG-ID>
+   ```
+
+   Where `<YOUR-ORG-ID>` is your organization ID, determined from the `viam organizations list` CLI command.
+   The full username you provide to your client should therefore resemble `db-user-abcdef12-abcd-abcd-abcd-abcdef123456`.
+
+For example, to connect to your Viam organization's MongoDB Atlas instance and query data using the `mongosh` shell:
+
+1. If you haven't already, [download the `mongosh` shell](https://www.mongodb.com/try/download/shell).
+   See the [`mongosh` documentation](https://www.mongodb.com/docs/mongodb-shell/) for more information.
+
+1. Run the following command to connect to the Viam organization's MongoDB Atlas instance from `mongosh`:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   mongosh "mongodb+srv://<YOUR-DB-HOSTNAME>" --apiVersion 1 --username db-user-<YOUR-ORG-ID>
+   ```
+
+   Where:
+
+   - `<YOUR-DB-HOSTNAME>` is your organization's assigned MongoDB Atlas instance hostname (including database name), determined from the [`viam data database hostname` CLI command](/services/data/configure-data-query/#configure-data-query).
+   - `<YOUR-ORG-ID>` is your organization ID, determined from the `viam organizations list` CLI command.
+     The full username you provide to the `--username` flag should therefore resemble `db-user-abcdef12-abcd-abcd-abcd-abcdef123456`.
+
+1. Once connected, you can run SQL or MQL to query captured data directly. For example:
+
+   - The following SQL query uses the MongoDB [`$sql` aggregation pipeline stage](https://www.mongodb.com/docs/atlas/data-federation/supported-unsupported/pipeline/sql/) to search the `sensorData` database and `readings` collection, and return sensor readings from an ultrasonic sensor on a specific `robot_id` where the recorded `distance` measurement is greater than `.2` meters:
+
+     ```mongodb {class="line-numbers linkable-line-numbers"}
```

Comment:
I don't know if this will work, let me know if not and I can change the javascript
```suggestion
     ```mongodb {class="command-line" data-prompt="$"}
```